README.md

```diff
@@ -128,6 +128,14 @@ These tests allow you to exercise any Ingest Node Pipelines defined by your pack
 
 _Context: package_
 
+#### System Tests
+
+These tests allow you to test a package's ability to ingest data end-to-end. 
+
+These tests will spin up an Elasticsearch node; a Kibana instance; a Package Registry; an Elastic Agent; and, depending on the package, an instance of the integration service for the package being tested. Then the tests will enroll the Agent with Fleet, configure a policy via Ingest Manager APIs, and assign the policy to the enrolled Agent. Once the Agent has received the policy, the tests will wait for the Agent to collect the appropriate data from the integration service and ingest it into the appropriate data stream in Elasticsearch.
```

Comment:
These semicolons looks a bit confusing to me, but I assume it's fine ;)

Reply:
Yes, they can be used when the list items themselves contain commas (see 3rd bullet point in https://writing.wisc.edu/handbook/grammarpunct/semicolons/).

Reply:
I changed this as well in ce611d2.

Reply:
Thank you for explanation and the guidance :)!